### PR TITLE
Cookies  setup + Amplitude integration ENG-913

### PR DESCRIPTION
**Pull request by DemolaJames**
# Task:

## Description

Add cookies setup to FE 

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## FE Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

## User Facing Checklist

- [ ] I fully understand the user problem this PR is solving
- [ ] I know who the target user is for this PR and have a deep understanding of that user
- [ ] I have tried this flow thinking from the pov of the target user for this PR
- [ ] I (or working w someone on team) have scheduled a user test for this PR (if it is a large change)

### CODE REVIEW

File: packages/frontend/src/hooks/useCookies/canStoreCookies.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { CookieNames } from "."
```
Comment (KMKoushik) [inline packages/frontend/src/hooks/useCookies/canStoreCookies.ts:1]: These are not hooks. Can you move all the functions to utils folder in `cookies.ts`?

File: packages/frontend/src/components/SettingsMenu.tsx [hunk 2]
```diff
@@ -216,6 +240,43 @@ const SettingMenu = () => {
           </List>
         </Box>
       </Modal>
+
+
+      <Modal
```
Comment (KMKoushik) [inline packages/frontend/src/components/SettingsMenu.tsx:245]: Can we make it a little bigger? It's so small right now?

File: packages/frontend/src/components/SettingsMenu.tsx [hunk 3]
```diff
@@ -78,6 +83,23 @@ const SettingMenu = () => {
...
 
+  const handleCookieModal = () => {
+    handleClose()
+    setOpenCookieModal((prevState) => !prevState)
+  }
+
+  const AcceptCookie = () => {
+    setCookieConsent(true);
+    setCookie(CookieNames.Consent, 'true')
+
```
Comment (KMKoushik) [inline packages/frontend/src/components/SettingsMenu.tsx:94]: Should we call initialize after accepting?

File: packages/frontend/src/components/SettingsMenu.tsx [hunk 4]
```diff
@@ -78,6 +83,23 @@ const SettingMenu = () => {
     setIsOver(isOver)
   }
 
+  const handleCookieModal = () => {
+    handleClose()
+    setOpenCookieModal((prevState) => !prevState)
+  }
+
+  const AcceptCookie = () => {
```
Comment (KMKoushik) [inline packages/frontend/src/components/SettingsMenu.tsx:91]: Should be `acceptCookie`

File: packages/frontend/src/hooks/useCookies/setCookie.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    const consent = (identifier == 'true') ? true : false
+    // save to cookie storage
+    Cookies.set(cookieName,identifier, {
+        expires: (consent) ? 365 : 10,
+        sameSite:'Strict',
+        path: '/'
+    });
+
+    if(consent){
+        initializeAmplitude()
```
Comment (KMKoushik) [inline packages/frontend/src/hooks/useCookies/setCookie.ts:16]: Oh okay, it's updated here!

**Review by KMKoushik — APPROVED** [review]

**Comment by alexisgauba** [comment]
Is there a way visually to make it a lot smaller? Feel like most sites just have a thing in the corner - looks pretty clunky right now 

Screen Shot 2022-11-17 at 9 31 29 PM

Screen Shot 2022-11-17 at 9 31 49 PM

Screen Shot 2022-11-17 at 9 26 51 PM

**Review by alexisgauba — CHANGES_REQUESTED** [review]
ui/ux needs to be improved

**Comment by tamtam3000** [comment]
Cookie box text needs to be changed to: We use cookies to support technical features that enhance your user experience and analyze frontend traffic. To learn more about these methods, including how to disable them, view our **Privacy Policy**. 

Note, _there is no separate cookie policy._ _Only a Privacy Policy_ that has an additional tracker / cookie section. Here is a link to the google doc with the final draft of the updated privacy policy: https://docs.google.com/document/d/1Y5Eg46AGbZJuOL0yNbD2g62uv39yi2Ppq47zeMg-9m4/edit. 

So remove the Cookie policy entirely from the frontend. 

**For the Privacy Policy text:** 

The Last Updated section needs to be updated to the date the PR is merged so, November ___ , 2022. 

Follow the google doc I sent that also has these notes. For example, in Section 4, the Analytics section, you need to add a link that will jump to a later section of the privacy policy page, where the text states, og to section 6.2 “Cookies the Interface Uses.”

**Privacy Policy Format:** 


Screenshot 2022-11-18 at 1 48 17 PM

The above section needs to be spaced out as in the google doc linked above so that each tracker type is in its own paragraph. I highlighted this in the docs so it's clear. 

6.2 Cookies the Interface Uses 

I updated the format in this section and text slightly. Please refer to the google doc linked above. 

Let me know if you have any questions on text or format related to the above.

**Pull request merged**